.github/workflows/linux.yml

```diff
@@ -331,3 +325,17 @@ jobs:
 
       - name: Show ccache stats
         run: ccache --show-stats
+
+  Overall_Status:
+    name: ci/gha_overall_status_linux
```

Comment:
It is not required as of now (as I've deleted the conditional start in the beginning of the file), but could be beneficial in the future.